docs/reference/sql/endpoints/cli.asciidoc

```diff
@@ -47,3 +43,27 @@ James S.A. Corey |Leviathan Wakes     |561            |1306972800000
 --------------------------------------------------
 // TODO it'd be lovely to be able to assert that this is correct but
 // that is probably more work then it is worth right now.
+
+The jar containing the SQL CLI is a stand alone Java application and
+the scripts just launch it. You can move it around to other machines
+without having to install Elasticsearch on them. Without the already
+provided script files, you can use a command similar to the following
+to start the SQL CLI. A Java version not earlier than the one used to 
+compile the SQL CLI Java files, needs to be installed to be able to run it:
+
+[source,bash]
+--------------------------------------------------
+$ ./java -jar [PATH_TO_CLI_JAR]/elasticsearch-sql-cli-[VERSION].jar https://some.server:9200
+--------------------------------------------------
+
+or
+
+[source,bash]
+--------------------------------------------------
+$ ./java -cp [PATH_TO_CLI_JAR]/elasticsearch-sql-cli-[VERSION].jar org.elasticsearch.xpack.sql.cli.Cli https://some.server:9200
+--------------------------------------------------
+
+The jar name will be different for each Elasticsearch version, thus
+the generic `VERSION` specified in the example above. Also, if not running
```

Comment:
Maybe also provide an example of the version? like `7.5.2` or something like that.